test/test_CMSSM_unitarity.cpp

```diff
@@ -0,0 +1,534 @@
+   Physical_input physical_input;
+   CMSSM_input_parameters input;
+   Spectrum_generator_settings settings;
+
+   // extract the input parameters from spectrum string
+   try {
+      slha_io.fill(settings);
+      slha_io.fill(qedqcd);
+      slha_io.fill(physical_input);
+      slha_io.fill(input);
+   } catch (const Error& error) {
+      BOOST_TEST_MESSAGE(error.what());
+      BOOST_TEST(false);
+   }
+
+   CMSSM_slha m(input);
+   slha_io.fill(m);
+   m.calculate_DRbar_masses();
+   m.reorder_DRbar_masses();
+
+   static constexpr int size = 36;
+
+   /*
+   Eigen::SparseMatrix<double> ref_matrix(size, size);
+   ref_matrix.insert(0, 0)   = -0.0046229614555045135;
+   ref_matrix.insert(0, 8)   = -0.0015409312401939398;
+   ref_matrix.insert(0, 18)  = -0.0021792849295675151;
+   ref_matrix.insert(0, 24)  = -0.0015391607421558148;
+   ref_matrix.insert(0, 29)  =  0.002191044670651851;
+   ref_matrix.insert(1, 1)   = -0.0030818624803878791;
+   ref_matrix.insert(2, 5)   = -0.0030819743036696748;
+   ref_matrix.insert(3, 6)   = -0.0021767019962289917;
+   ref_matrix.insert(4, 7)   =  0.0030986050890011384;
+   ref_matrix.insert(8, 8)   = -0.0046226259917450515;
+   ref_matrix.insert(8, 18)  = -0.00217920594520562;
+   ref_matrix.insert(8, 24)  = -0.001539104896783423;
+   ref_matrix.insert(8, 29)  =  0.0021909651729722815;
+   ref_matrix.insert(9, 12)  = -0.0030818626029138676;
+   ref_matrix.insert(9, 28)  = -0.0028428770609625234;
+   ref_matrix.insert(10, 13) = -0.0021766230189459593;
+   ref_matrix.insert(10, 32) = -0.0028428770609625234;
+   ref_matrix.insert(11, 14) =  0.0030984926623045142;
+   ref_matrix.insert(11, 19) = -0.0028428770609625234;
+   ref_matrix.insert(12, 25) = -0.0028428770609625234;
+   ref_matrix.insert(13, 17) = -0.0028428770609625234;
+   ref_matrix.insert(14, 23) = -0.0028428770609625234;
+   ref_matrix.insert(15, 30) = -0.0030819743036696761;
+   ref_matrix.insert(16, 31) =  0.0030986050740594282;
+   ref_matrix.insert(17, 32) = -0.00092412316568786881;
+   ref_matrix.insert(18, 18) = -0.0061639486073393496;
+   ref_matrix.insert(18, 24) =  0.0030986050740594282;
+   ref_matrix.insert(18, 29) = -0.00092412316568786881;
+   ref_matrix.insert(19, 23) =  0.0030986050740594282;
+   ref_matrix.insert(20, 27) = -0.00092412316568786881;
+   ref_matrix.insert(21, 33) = -0.0043768955684380072;
+   ref_matrix.insert(22, 34) = -0.0040419594819556121;
+   ref_matrix.insert(24, 24) = -0.0087537911368760126;
+   ref_matrix.insert(24, 29) = -0.0040419594819556121;
+   ref_matrix.insert(25, 28) = -0.0040419594819556121;
+   ref_matrix.insert(26, 35) = -0.0031153256164892489;
+   ref_matrix.insert(29, 29) = -0.006230651232978497;
+   */
+
+   auto unitarityMatrix = CMSSM_unitarity::max_scattering_eigenvalue_infinite_s(m);
+   for (int i = 0; i < size; i++) {
+      for (int j = 0; j < size; j++) {
+         //BOOST_CHECK_CLOSE_FRACTION(unitarityMatrix.second.coeff(i,j), ref_matrix.coeff(i, j), 1e-16);
```

Comment:
It seems this test does not test anything. Do you want to postpone this?

Reply:
I'd leave it for now. I'll make a mark in the FS 3.0 manual that we want to showcase this calculation in the CMSSM and then I'll fix this test to match published example.

Reply:
Ok. Please also mark the code in the test with `// @todo(wkotlarski)` (so we can grep for it).

Reply:
Good idea. Is this a syntax we should always follow in the code? `// @todo(GITHUB_USERNAME)`?

Reply:
Yes, I'd vote for that. :)